installing/disconnected_install/installing-oc-mirror-v2.adoc

```diff
@@ -0,0 +1,53 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="installing-oc-mirror-v2"]
+= Mirroring images for a disconnected installation using the V2
+include::_attributes/common-attributes.adoc[]
+:context: installing-mirroring-disconnected
+
+toc::[]
+
+:FeatureName: oc-mirror v2
+include::snippets/technology-preview.adoc[]
+
+Running your cluster in a restricted network without direct internet connectivity is possible by installing the cluster from a mirrored set of {product-title} container images in a private registry. This registry must be running at all times provided that the cluster is running. See the link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.15/html/installing/disconnected-installation-mirroring#prerequisites_installing-mirroring-disconnected[Prerequisites] section for more information.
+
+You can use the oc-mirror OpenShift CLI (`oc`) plugin to mirror images to a mirror registry in your fully or partially disconnected environments. You must run oc-mirror from a system with internet connectivity in order to download the required images from the official Red Hat registries.
+
```

Comment:
Perhaps this paragraph here? https://github.com/openshift/openshift-docs/pull/76917/files#diff-5e72668c367f748ce622ac7f5914e485b7c7a4803986b5ddfd954fda7663a218R49